scripts/semconv/generate.sh

```diff
@@ -5,7 +5,11 @@ ROOT_DIR="${SCRIPT_DIR}/../../"
 
 # freeze the spec version to make SpanAttributess generation reproducible
 SPEC_VERSION=v1.7.0
-GENERATOR_VERSION=0.7.0
+GENERATOR_VERSION=0.8.0
```

Comment:
Updated to the latest version of the build tool -- seems to be working ok.
Still using the same Spec version though.

Reply:
According to https://hub.docker.com/r/otel/semconvgen/tags latest version is `0.23.0` but it's better not to upgrade. I've tried myself and introduces some api changes that produces a different output. Basically with newer versions you need to:
- point to the root path for all models
- use `--only` cli option to pick what models you want

See how java agent is using it
https://github.com/open-telemetry/semantic-conventions-java/blob/2be178a7fd62d1073fa9b4f0f0520772a6496e0b/build.gradle.kts#L108

So when we plan to upgrade we should take this into account

Reply:
Yeah, I was trying to limit / stop any unexpected breaking changes with this change.

And as part of upgrading to the current semantic conventions we are most likely going to need to jump. As a side note, I'm also working on updating the build-tools to support the upcoming log event semantic conventions (I'm guessing it's going to be a while before it gets accepted and included in a tools release though)